.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,37 @@
+name: 機能要望
+description: OreOreBot2の機能の要望を行う
+labels: ["feature request", "enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        🚀 機能要望の提出ありがとうございます
+  - type: checkboxes
+    id: confirmation
+    attributes:
+      label: 機能要望の提出を行う前に
+      description: 機能要望の提出を行う前に以下の項目を確認してください
+      options:
+        - label: すでに[同じ機能要望を行うIssue](https://github.com/approvers/OreOreBot2/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature+request%22)が存在しないか確認しましたか?

```

```suggestion
        - label: すでに[同じ機能要望を行うIssue](https://github.com/approvers/OreOreBot2/issues?q=)が存在しないか確認しましたか?
```
Closed で未リリースなどこのフィルタでは見つからないかもしれないので, 外しておいてよいかと. バグ報告の方も同じです